tests/cupy_tests/core_tests/test_ndarray.py

```diff
@@ -63,6 +63,13 @@ def test_order(self):
         self.assertTrue(a.flags.f_contiguous)
         self.assertTrue(not a.flags.c_contiguous)
 
+    def test_order_None(self):
+        a = core.ndarray(self.shape, order=None)
+        a_cpu = numpy.ndarray(self.shape, order=None)
+        self.assertTupleEqual(a.strides, a_cpu.strides)
+        self.assertTrue(a.flags.c_contiguous)
+        self.assertTrue(not a.flags.f_contiguous)
```

Comment:
Could you add assertions to test `a_cpu`'s `c_contiguous` and `f_contiguous` as well? That way we can be notified when NumPy changes the behavior.

Reply:
Are you okay with switching to test `a.flags.c_contiguous == a_cpu.flags.c_contiguous` and similarly for `'F'`? I guess the purpose is to track Numpy behavior.

Reply:
Yes, that's also good.